README.md

```diff
@@ -1,5 +1,8 @@
-# LDSA Portal [![docs](https://readthedocs.org/projects/ldsa-portal/badge/?version=latest&style=plastic)](https://ldsa-portal.readthedocs.io/en/latest/)
+# LDSA Portal
+<a href="https://github.com/psf/black"><img alt="Code style: black" src="https://img.shields.io/badge/code%20style-black-000000.svg"></a>
+<a href="https://ldsa-portal.readthedocs.io/en/latest/"><img alt="Code style: black" src="https://readthedocs.org/projects/ldsa-portal/badge/?version=latest&style=plastic"></a>
```

Comment:
For consistency, could you keep this as Markdown?

Reply:
Done